samza-core/src/main/java/org/apache/samza/table/remote/TableWriteFunction.java

```diff
@@ -44,13 +43,19 @@
 public interface TableWriteFunction<K, V> extends Serializable, InitableFunction, ClosableFunction {
   /**
    * Store single table {@code record} with specified {@code key}. This method must be thread-safe.
+   *
+   * The key is deleted, if record is {@code null}.
```

Comment:
nit: add "its corresponding" to match the previous doc

Reply:
There is only one key/value pair, hence no mentioning.